bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types.go

```diff
@@ -279,7 +279,7 @@ type User struct {
 
 	// Passwd specifies a hashed password for the user
 	// +optional
-	Passwd *string `json:"passwd,omitempty"`
+	HashedPasswd *string `json:"hashedPasswd,omitempty"`
```

Comment:
Renaming an existing field is an API-breaking change and it cannot be implemented without following [deprecation guidelines](https://github.com/kubernetes-sigs/cluster-api/blob/main/CONTRIBUTING.md#apis).

Also, given my understanding of the [cloud-init code](https://github.com/canonical/cloud-init/blob/bae9b11da9ed7dd0b16fe5adeaf4774b7cc628cf/cloudinit/distros/__init__.py), `passwd` is used for setting the password of new users, which is a use case we want to continue to support.

Reply:
ok,I will have some modify